src/components/Product.jsx

```diff
@@ -0,0 +1,49 @@
+import PropTypes from 'prop-types';
+import {
+    displayProductTitle,
+    formatProductPrice,
+    displayAvailableScreenSizes, productOutOfStock,
+} from '../helpers/helperFunctions';
+
+import checkIcon from '../assets/check.png';
+import minusIcon from '../assets/minus.png';
+import '../styles/Product.css';
+import outOfStock from "../assets/out-of-stock.png";
+
+const Product = ({ product }) => {
+    const productOptions = (options) => {
+        return (
+            <div>
+                {options.map((option, index) => (
+                    <span key={index} className="product-option-wrapper">
+                        {option.name}
+                        <img className='product-option-image'
+                             src={option.applicable ? `${checkIcon}` : `${minusIcon}`}
+                             alt={option.applicable ? 'check' : 'min'}
+                        />
+                    </span>
+                ))}
+            </div>
+        );
+    };
+
+    return (
+        <div className="product-container">
```

Comment:
In de App.jsx file maak je keurig gebruik van meerdere semantische elementen, echter gebruik je bij de componenten voornamelijk divjes. Let er op dat je ook hier gebruik maakt van semantische elementen.